src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -80,7 +80,7 @@ def _create_session(
         s.auth = auth if auth else s.auth
         s.proxies = proxies if proxies else s.proxies
 
-        max_retries = self.builtin.convert_to_integer(max_retries)        
+        max_retries = int(max_retries)        
```

Comment:
Do you have any failing scenario/example test without using ``self.builtin.convert_to_integer(max_retries) ``?